omod-common/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/MainResourceController.java

```diff
@@ -152,7 +152,7 @@ public Object delete(@PathVariable("resource") String resource, @PathVariable("u
 	 * @param response
 	 * @throws Exception
 	 */
-	@RequestMapping(value = "/{resource}/{uuid}", method = RequestMethod.DELETE, params = "purge")
+	@RequestMapping(value = "/{resource}/{uuid}", method = RequestMethod.DELETE, params = "purge=true")
```

Comment:
There you go! See how you didn't have to make any changes within the controller method? And everything still works